src/flow/assembly_mh.hh

```diff
@@ -793,12 +781,11 @@ protected:
         double conduct =  ad_->conductivity.value(ele.centre(), ele);
 	    double bet =  ad_->beta.value(ele.centre(), ele);
 	    auto w = ad_->field_ele_velocity.value(ele.centre(), ele);
-        double scale = 1 / cs /conduct;
+        arma::mat33 scale = 1 / cs /conduct * arma::eye(3,3) + bet*((arma::kron(w,w) / arma::norm(w)) + arma::norm(w)*arma::eye(3,3) ) / cs;
```

Comment:
There should be inverse anisotropy in the first term, similarly as in AssemblyMH.